src/PipeLineFactory/bin/trigger.ts

```diff
@@ -8,7 +8,7 @@ const stageDev = { account: '928065939415', region: 'eu-west-1' };
 
 const projectName : string = "PipeLine-Factory"
 const factoryProperties : FactoryProperties = {
-    githubRepositoryBranch : "master",
+    githubRepositoryBranch : "feat/spx-116",
```

Comment:
Do you mean to tie this to a feat branch?

Reply:
transitionally while I test only. before merging I will change it to master. Thanks for the comment , it will remind of doing so